packages/core/src/hooks/useActiveDescendantListFocus/__stories__/useActiveDescendantListFocus.mdx

```diff
@@ -19,7 +19,7 @@ import * as UseActiveDescendantListFocusStories from "./useActiveDescendantListF
 ### Import
 
 ```js
-import { useActiveDescendantListFocus } from "monday-ui-react-core/hooks";
+import { useActiveDescendantListFocus } from "@vibe/core/hooks";
```

Comment:
do we want to have it under @vibe/hooks?

Reply:
I'm all for it but it's not a blocker for the release